### PR TITLE
Feature/fix code

### PR DESCRIPTION
**Pull request by JessMotta**
Updates:
- Shows the category/categories of mentors on cards
- Search mentors by category and return all that have part of the searched word
- If the category doesn't exist, a message is displayed on the screen
- If there is no mentor in the category, a warning message is displayed on the screen
- The mentor profile shows the category/category and the rating associated with it
- Calendar problem solved

### CODE REVIEW

File: scripts/api.js [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
 
-//   // Cria um usuário e o retorna 
-//   console.log('Cria Usuário, parâmetros: {email: string; name: string; password: string;} >>>', await createUser({
-//     name: 'Afonso2',
-//     email: 'afons2o@gmail.com',
-//     password: '12345678'
-//   }));
-// });
+  // Cria um usuário e o retorna 
+  console.log('Cria Usuário, parâmetros: {email: string; name: string; password: string;} >>>', await createUser({
```
Comment (luanaggoncalves) [inline scripts/api.js:24]: Opa, acho que podemos remover esses console.log() já que não são mais necessários

File: scripts/mentorProfile.js [hunk 2]
```diff
@@ -25,30 +25,61 @@ async function initializeInfo() {
...
-  skill.textContent = dataMentors.skills.skill
-  
+  category_1.textContent = dataMentors.Categories[0].name;
+  category_1Rating.textContent = dataMentors.Categories[0].name;
+
+ 
+  // PARA CATEGORIA 1
   // mostrar a avaliação da habilidade
-  let rating = parseFloat(dataMentors.skills.rating);
+  let rating_1 = dataMentors.Categories[0].UserCategories.rating;
```
Comment (luanaggoncalves) [inline scripts/mentorProfile.js:50]: Podemos usar const, já que esse varíavel não é reassignada.

```suggestion
  const rating_1 = dataMentors.Categories[0].UserCategories.rating;
```

File: scripts/register.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+// document.addEventListener("DOMContentLoaded", () => {
+//   initializeForm();
+// });
```
Comment (luanaggoncalves) [inline scripts/register.js:3]:
```suggestion
```

**Review by luanaggoncalves — CHANGES_REQUESTED** [review]

File: css/cards.css [hunk 4]
```diff
@@ -1,25 +1,28 @@
 .card {
   background-color: var(--color-white);
   width: 234px;
+  /* max-width: fit-content; */
```
[the remark this answers is not in the document]
Comment (JessMotta) [reply css/cards.css:4]: Removed

File: pages/register.html [hunk 5]
```diff
@@ -68,7 +68,7 @@ <h1><img src="../assets/images/logo.png" alt=""></h1>
 
                     </div>
 
-                    <div class="register__specialty">
+                    <!-- <div class="register__specialty">
```
[the remark this answers is not in the document]
Comment (JessMotta) [reply pages/register.html:71]: removed

File: scripts/api.js (hunk 1 above)
Comment (JessMotta) [reply scripts/api.js:24]: removed

File: scripts/mentorProfile.js (hunk 2 above)
Comment (JessMotta) [reply scripts/mentorProfile.js:50]: fixed

File: scripts/register.js (hunk 3 above)
Comment (JessMotta) [reply scripts/register.js:3]: removed

**Pull request merged**